modules/core/src/main/scala/org/tessellation/domain/cell/L0Cell.scala

```diff
@@ -85,10 +94,18 @@ object L0Cell {
 
   object Coalgebra {
 
-    def processDAGL1[F[_]: Async](data: Signed[L1Output]): CoalgebraR[F] = {
+    def processDAGL1[F[_]: Async: Logger](data: Signed[L1Output], collateral: Collateral[F]): CoalgebraR[F] = {
       def res: StackF[CoalgebraCommand] = Done(AlgebraCommand.EnqueueDAGL1Data(data).asRight[CellError])
-
-      res.pure[F]
+      def errorRes: F[StackF[CoalgebraCommand]] =
+        Logger[F]
+          .debug(s"Peers which signed block sent by ${data.proofs.head.id} have not sufficient collateral")
+          .as(Done(CellError("Peers which signed block have not sufficient collateral").asLeft[Ω]))
```

Comment:
I'm not sure if rejecting blocks in Cell is necessary. Maybe it would be enough to reject them only during L0 consensus? 🤔